src/id.md

```diff
@@ -58,7 +58,7 @@ Using the account you wish to verify, go under the Developer tab and follow thes
 
 - Click Extrinsics.
 - Select `Identity` and `requestJudgement`.
-- Under `regIndex`, place `1` (`reg_index` = `0` is no longer active). Registrar `1` has a fee of 500 `XOR`.
+- Under `regIndex`, place `1` (`reg_index` = `0` is no longer active). Registrar `1` has a fee of 5000 `XOR`.
```

Comment:
Why is the fee here 5k, but in the intro its 5600?

Reply:
Total fee includes the transcation fee. Which is around the 600 XOR mark